python/ray/tests/test_actor.py

```diff
@@ -836,5 +836,35 @@ def get_value(self):
         actor_create_time - start, end - start))
 
 
+@pytest.mark.skip(
+    "This test fails. The actor creation task doesn't actually release "
+    "resources")
+def test_release_cpus_when_actor_creation_task_blocking(shutdown_only):
+    ray.init(num_cpus=1)
```

Comment:
I changed this test to use 1 CPU instead of 2 because I don't think it was actually testing the intended behavior if we set it to 2. Note that it fails with 1, which is why I'm skipping it. Probably a scheduler bug.

Also @mfitton this is an example of a failure that should be diagnosable through the dashboard, but it currently is not (we are waiting for a task that never gets executed because of a lack of resources.